internal/tools/document-lint/mdparse/diff/diff.go

```diff
@@ -0,0 +1,396 @@
+package diff
```

Comment:
diff seems like the wrong name here, maybe check? as we are not diffing so much as checking various things